api/v1/kubtest.yaml

```diff
@@ -1,23 +1,19 @@
 openapi: 3.0.1
 
 info:
-  title: kubtest
-  description: 'Efficient testing of k8s applications mandates a k8s native approach to test mgmt/definition/execution - kubtest provides a “quality control plane” that natively integrates testing activities into k8s development and operational workflows'
-  termsOfService: http://kubtest.io/terms/
+  title: Kubtest API
+  description: 'Kubtest provides a Kubernetes-native framework for test definition, execution and results'
   contact:
-    email: api@kubtest.io
+    email: kubtest@kubshop.io
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
-  version: 0.0.5
+  version: 1.0.0
```

Comment:
@olensmar can we put here tag ?

Reply:
or take it somehow when building (maybe some placeholder ? don't know what possibilities are there

Reply:
yeah - not sure either... let's go with this for now